test/mongo_util.py

```diff
@@ -1,23 +1,106 @@
 import logging
 import os
-import uuid
-from pathlib import Path
-from typing import Tuple
+
+from configparser import ConfigParser
 from datetime import datetime
+from pathlib import Path
+from typing import List, Tuple, Union
+
 from mongo_controller import MongoController
 from AbstractHandle.Utils import MongoUtil
 
-MONGO_EXE_PATH="MONGO_EXE_PATH"
-MONGO_TEMP_DIR="MONGO_TEMP_DIR"
-
-
-def get_mongo_info() -> Tuple[Path, Path]:
+# Mongo config
+TEST_CONFIG_PATH = "HANDLE_SERVICE_TEST_CFG"
+TEST_SECTION = "HandleService2"
+TEST_MONGO_EXE = "test.mongo.exe"
+TEST_TEMP_DIR = "test.temp.dir"
+TEST_USE_WIRED_TIGER = "test.mongo.wired_tiger"
+TEST_DELETE_TEMP_DIR = "test.delete.temp.dir"
+
+# Deploy config
+TEST_AUTH_SERVICE_URL = "auth-service-url"
+TEST_AUTH_URL = "auth-url"
+TEST_SHOCK_URL = "shock-url"
```

Comment:
```suggestion
TEST_BLOBSTORE_URL = "blobstore-url"
```

Not going to do any more of these, just look for shock and replace it with blobstore where it makes sense

Not in the deploy.cfg thought, that'd mean we've have to change the config in rancher

Reply:
👍